Alpaca.Markets/DataStreamingClientBase.cs

```diff
@@ -41,7 +41,7 @@ public IEnumerable<String> Streams
                              ?? throw new RestClientErrorException());
 
         public void OnUpdate(
-            Boolean isSubscribed) => Subscribed = isSubscribed;
+            Boolean isSubscribed) => Subscribed = DateTime.UtcNow.Ticks;
```

Comment:
Another breaking change - now in logic. The existing `Subscribed` property has 2 states and its value changed on each subscription update message received from the server. In your case, this property will return the time of the last subscription message received even if this message has no relation to this particular subscription.